packages/web-react/src/components/Accordion/AccordionContent.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Collapse } from '../Collapse';
+import { AccordionContentProps } from '../../types';
+import { useAccordionStyleProps } from './useAccordionStyleProps';
+import { useAccordionContext } from './AccordionContext';
+import { useAccordionItemContext } from './AccordionItemContext';
+import { useAccordionAriaProps } from './useAccordionAriaProps';
+
+const AccordionContent: React.FC<AccordionContentProps> = ({ children }) => {
+  const { classProps } = useAccordionStyleProps();
+  const { isItemOpen, open } = useAccordionContext();
+  const { id } = useAccordionItemContext();
+
+  const isOpen = React.useMemo(
+    () => (id && isItemOpen && open ? isItemOpen(open, id) : false),
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [open, id],
+  );
+
+  const { contentProps } = useAccordionAriaProps({ id, isOpen });
+
+  return (
+    <Collapse isOpen={isOpen} {...contentProps}>
```

Comment:
Missing `styleProps` here.

Reply:
This is only a sub component. We once solved something similar, but in the end we left styleProps only for the main and wrapping components.

Reply:
Yeah, I know we discussed this earlier. In this case if we allow the addition of class to the root component we will make it worse to style inner items. Image when we have Accordion with 10 items and we want to style the 5th item header and 7th item content differently. If we allow `UNSAFE_` props for subcomponents it will be much simpler to do that.

I have also consulted this with @adamkudrna if this case is correct and he suggested doing so.

Reply:
done